### DISCUSSION TITLE
Phase 3: integration

### DISCUSSION THREAD

**Discussion by mflatt** [racket/rhombus#521] [Announcements]
We're now officially in the "integration" phase for Rhombus.

This repo has been renamed to `racket/rhombus`, but `racket/rhombus-prototype` redirects here. The repo content has been split into multiple packages, but the `rhombus-prototype` package (not meant to be used going forward) implies the initial set of Rhombus packages, for your transition convenience.

In this new phase, we start building out Rhombus libraries. That will include documentation, and Rhombus Scribble does exist (and is used to document the Rhombus language). But Rhombus Scribble is not itself documented, and I'd like to take the opportunity to further improve documentation support and use it to document Rhombus Scribble in the coming days or weeks.

I expect that we'll want to create a list of libraries to be adapted/ported. Let's discuss that at the July 25 meeting (#180).

**Comment by mflatt** [comment]
We'll use this [wiki page](../wiki/Rhombus-Integration-Tasks) to coordinate integration tasks.

**Reply by wrq** [reply]
Here’s hoping this wasn’t a monumental waste of time just to implement a syntax layer nobody will use! 🤞

**Reply by spdegabrielle** [reply]
Some people are already using it, but it seems we need to start showcasing examples

**Reply by wrq** [reply]
In school projects and demos and things, sure. I just find it astounding that all these PhD hours get spent on something that amounts to little more than shaped macros, is all. I find it very discouraging that Racket isn’t used more in industry, and even more-so that so many steering or otherwise significant contributors are working on stuff like this rather than courting commercial adoption. 

At the end of the day, I’ll probably try it out, have fun, and do nothing with it. Like basically everybody else…

**Reply by sorawee** [reply]
Umm. Please be considerate? How would you feel if a stranger knocks on your door and says they hope your life "wasn’t a monumental waste of time"? Concerns are certainly appreciated, but there's no need to be dismissive.

But to answer to your points, Rhombus development is Racket development. By trying to implement it, we learned what are Racket's shortcomings, and we already improved Racket by adding missing features back to Racket, so that people are able to do language-oriented programming even better in Racket. 

It's also not an either-or. Developing Rhombus doesn't mean we can no longer promote Racket.

**Reply by benknoble** [reply]
I finally figured out why the link appears broken for me. The original text is

```
We'll use this [wiki page](../wiki/Rhombus-Integration-Tasks) to coordinate integration tasks.
```

On a web browser, this takes me to https://github.com/racket/rhombus/wiki/Rhombus-Integration-Tasks as expected.

On the GitHub iOS app, however, this goes to https://github.com/wiki/Rhombus-Integration-Tasks, which obviously doesn't exist. This is definitely a problem with GitHub's app, but perhaps the link could be edited to be absolute? (I recognize that relative links are helpful for renames, though, so feel free to leave as-is.)
